keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -921,18 +1008,22 @@ measured in seconds.  The units change: `mu_single` and `mu_i` are no
 longer in bits per second but rather are in service-seconds per
 second; we call this unit "seats" for short.  We now say `mu_single`
 is 1 seat.  As before: when it is time to dispatch the next request in
-the real world we pick from the queue whose current packet
-transmission would finish soonest in the virtual world, using
-round-robin ordering to break ties.
+the real world we pick from a queue with a request that will complete
+soonest in the virtual world, using round-robin ordering to break
+ties.
 
 ##### Not knowing service duration up front
 
 The final change removes the up-front knowledge of the service
 duration of a request.  Instead, we use a guess `G`.  When a request
 finishes execution in the real world, we learn its actual service
-duration `D`.  At this point we adjust the explicitly represented B
-and E (and F, if using those) values of following requests in that
-queue to account for the difference `D - G`.
+duration `D`.  At this point we adjust the expected end time of the
+request in the virtual world.  In the case where that estimate end
+time equals the current time (it will never precede), the request
```

Comment:
Can you explain why it will never precede?

Reply:
How about the following?

> ... (it will never precede, because a packet is never transmitted faster in the virtual world than in the real world --- `mu(i,t) <= mu_single`) ...

Reply:
The latest revision attempts to straighten this out.